src/vendor/webauthn.js

```diff
@@ -0,0 +1,185 @@
+"use strict";
+define(function () {
+
+  var webauthn = (function () {
+
+    const webauthnDB = (function () {
```

Comment:
we can remove the webauthnDB and related functions and references to these.

We just need the makeCredential() and getAssertion() methods.